regress/run_test.pl

```diff
@@ -970,6 +970,14 @@ sub run_loader_and_check_output
 			}
 		}
 
+		if ( $loader_options =~ m{^-Z } ) {
```

Comment:
Does this not only match `-Z` if given as the very first option ?

Reply:
Yes. I propose we replace that with `m{^-Z|\s-Z\s}` to cover all the valid locations:

```bash
echo -e "-Z a b\na -Z b\na b -Z\na b c\na-Z b c\na b c-Z" | perl -ne 'print if m{^-Z|\s-Z\s};'
```

output:
```
-Z a b
a -Z b
a b -Z
```